src/data_snack/connections/memcached.py

```diff
@@ -11,15 +11,23 @@ class MemcachedConnection(Connection):
     def get(self, key: Text) -> bytes:
         return self.connection.get(key)
 
-    def set(self, key: Text, value: Text) -> bool:
-        return self.connection.set(key, value)
+    def set(self, key: Text, value: Text, expire: int = 0) -> bool:
+        return self.connection.set(key, value, expire=expire)
+
+    def delete(self, key: Text) -> bool:
+        return self.connection.delete(key, noreply=False)
```

Comment:
Due to this: _"If noreply is True (or if it is unset and self.default_noreply is True), the return value is always True. Otherwise returns True if the key was deleted, and False if it wasn’t found."_ (https://pymemcache.readthedocs.io/en/latest/apidoc/pymemcache.client.base.html#pymemcache.client.base.Client.delete)